packages/replay/src/replay.ts

```diff
@@ -60,6 +54,12 @@ export class ReplayContainer implements ReplayContainerInterface {
    */
   public recordingMode: ReplayRecordingMode = 'session';
 
+  /** These are here so we can overwrite them in tests etc. */
+  public timeouts = {
+    sessionIdle: SESSION_IDLE_DURATION,
+    maxSessionLife: MAX_SESSION_LIFE,
+  };
```

Comment:
Is there a way we only allow writes to these timeouts during tests? I worry some crafty people may change these values to something unsupported.

Reply:
you'd have to reach into a private object with no docs whatsover 😅 (e.g. `replay._replay.timeouts`). I'll add `@hidden`, actually. Other than this, not sure 😅 but at this point people could be overwriting anything in there basically xD

Reply:
Could we make this into getter/setter and have the setters check for an env var that's set during test? Maybe you're right though it's probably overkill

Reply:
IMHO if we do that users can still overwrite it as well if they really want 😅 and we'd have to look this up on `window` etc. which IMHO would also be a bit weird, so I'd leave it like this for now. If we find this is abused we can still change it later, I'd say.

Reply:
I've made this `readonly` & `const`, hopefully making it clear that this cannot/should not be overwritten!